main.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import FastAPI, Depends, Query
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from crud import get_author_from_db, get_all_books, create_book_db
```

Comment:
The import statement for `get_author_from_db`, `get_all_books`, and `create_book_db` from `crud` is redundant since `crud` is already imported. You can access these functions using `crud.get_author_from_db`, `crud.get_all_books`, and `crud.create_book_db`.